src/lang/util.ts

```diff
@@ -1,45 +1,7 @@
 import { Selections } from 'lib/selections'
-import { Program, PathToNode } from './wasm'
-import { getNodeFromPath } from './queryAst'
+import { PathToNode } from './wasm'
 import { ArtifactGraph, filterArtifacts } from 'lang/std/artifactGraph'
 import { isOverlap } from 'lib/utils'
-import { err } from 'lib/trap'
-
-export function pathMapToSelections(
```

Comment:
This function was unused. Instead of updating it, I deleted it. But if it's useful to someone, I can try to update it.